BUILDING.txt

```diff
@@ -57,7 +57,7 @@ Refer to  dev-support/docker/Dockerfile):
 
 * Open JDK 1.8
   $ sudo apt-get update
-  $ sudo apt-get -y install java-8-openjdk
+  $ sudo apt-get -y install openjdk-8-jdk
```

Comment:
this is in fact the package we use in the Dockerfile build script. https://github.com/apache/hadoop/blob/trunk/dev-support/docker/pkg-resolver/packages.json#L264